<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_Data_Change</class>
 <widget class="QWidget" name="DB_Data_Change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="change_tableView">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>50</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changepaper_dele_button">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_change_button">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>原姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="old_name_line"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>新姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="new_name_line">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
